src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -1582,16 +1581,12 @@ private void searchExpand(final Menu menu, final MenuItem searchItem) {
     beforeSearchComposeVisibility = composePanel.getVisibility();
     composePanel.setVisibility(View.GONE);
 
-    beforeSearchAttachVisibility = attachmentManager.getVisibility();
-    attachmentManager.setVisibility(View.GONE);
```

Comment:
apparently calling this causes `AttachmentManager.attachmentViewStub.resolved()` to be `true` then if you never attached an image before `inflateStub()` was not called and all the views are `null` and will not be initialized since `inflateStub()` only initialize them if `attachmentViewStub.resolved() == false`

Reply:
thanks for explanation, indeed, i was wondering, at a first glance :)